script.js

```diff
@@ -0,0 +1,240 @@
+        { code: 221, en: ']', ru: 'ъ', printable: true },
+        { code: 220, en: '\\', ru: '\\', printable: true },
+        { code: 46, en: 'DEL', ru: 'DEL', printable: false },
+    ],
+
+    [
+        { code: 20, en: 'Caps Lock', ru: 'Caps Lock', printable: false },
+        { code: 65, en: 'a', ru: 'ф', printable: true },
+        { code: 83, en: 's', ru: 'ы', printable: true },
+        { code: 68, en: 'd', ru: 'в', printable: true },
+        { code: 70, en: 'f', ru: 'а', printable: true },
+        { code: 71, en: 'g', ru: 'п', printable: true },
+        { code: 72, en: 'h', ru: 'р', printable: true },
+        { code: 74, en: 'j', ru: 'о', printable: true },
+        { code: 75, en: 'k', ru: 'л', printable: true },
+        { code: 76, en: 'l', ru: 'д', printable: true },
+        { code: 186, en: ';', ru: 'ж', printable: true },
+        { code: 222, en: '\'', ru: 'э', printable: true },
+        { code: 13, en: 'Enter', ru: 'Enter', printable: false },
+    ],
+
+    [
+        { code: 16, en: 'Shift', ru: 'Shift', printable: false },
+        { code: 226, en: '\\', ru: '\\', printable: true },
+        { code: 90, en: 'z', ru: 'я', printable: true },
+        { code: 88, en: 'c', ru: 'с', printable: true },
+        { code: 67, en: 'v', ru: 'м', printable: true },
+        { code: 86, en: 'b', ru: 'и', printable: true },
+        { code: 66, en: 'n', ru: 'т', printable: true },
+        { code: 78, en: 'm', ru: 'ь', printable: true },
+        { code: 77, en: ',', ru: 'б', printable: true },
+        { code: 188, en: '.', ru: 'ю', printable: true },
+        { code: 190, en: '\/', ru: '.', printable: true },
+        { code: 191, en: '\'', ru: '.', printable: true },
+        { code: 38, en: '▲', ru: '▲', printable: true },
+        { code: 16, en: 'Shift', ru: 'Shift', printable: false },
+    ],
+
+    [
+        { code: 17, en: 'Ctrl', ru: 'Ctrl', printable: false },
+        { code: 91, en: 'Win', ru: 'Win', printable: false },
+        { code: 18, en: 'Alt', ru: 'Alt', printable: false },
+        { code: 32, en: ' ', ru: ' ', printable: true },
+        { code: 18, en: 'Alt', ru: 'Alt', printable: false },
+        { code: 37, en: '◄', ru: '◄', printable: true },
+        { code: 40, en: '▼', ru: '▼', printable: true },
+        { code: 39, en: '►', ru: '►', printable: true },
+        { code: 17, en: 'Ctrl', ru: 'Ctrl', printable: false }
+    ],
+];
+
+let lang = 'en';
+
+fillKeyboard();
+
+function changeLanguage() {
+
+    lang = (lang === 'ru') ? 'en' : 'ru';
+    //console.log(lang);
+    fillKeyboard();
+}
+
+
+function fillKeyboard() {
+
+    for (let i = 0; i < keyboard.children.length; i++) {
+        for (let j = 0; j < keyboard.children[i].children.length; j++) {
+            keyboard.children[i].children[j].textContent = buttons[i][j][lang];
+        }
+    }
+}
+
+
+
+const body = document.querySelector('body')
+
+body.addEventListener('keydown', (event) => {
+    buttons.forEach((i, indexI) => {
```

Comment:
if you implement id = keyCode for all buttons, you'll not need to use forEach to find button.